MaxMind.MinFraud.UnitTest/WebServiceClientTest.cs

```diff
@@ -157,8 +152,8 @@ public async Task Test200WithNoBody()
 
             var exception = await Record.ExceptionAsync(async () => await client.InsightsAsync(request));
             Assert.NotNull(exception);
-            Assert.IsType<HttpException>(exception);
-            Assert.Contains("Received a 200 response for minFraud Insights but there was no message body",
+            Assert.IsType<MinFraudException>(exception);
```

Comment:
This exception changed as we are using the combined extension method `ReadFromJsonAsync` for reading the content and deserializing it. I don't think the change is matters.